ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-popover.js

```diff
@@ -28,8 +29,10 @@ const AdvancedGasFeePopover = () => {
         onClose={closeAllModals}
         footer={<AdvancedGasFeeSaveButton />}
       >
-        <Box className="advanced-gas-fee-popover__wrapper">
+        <Box className="advanced-gas-fee-popover__wrapper" margin={4}>
           <AdvancedGasFeeInputs />
+          <div className="advanced-gas-fee-popover__separator" />
+          <AdvancedGasFeeDefaults />
```

Comment:
I am not sure if this margin is really needed, inputs already have margins. Can you plz check margins are not doubled:
<img width="363" alt="Screenshot 2021-12-03 at 12 11 25 PM" src="https://user-images.githubusercontent.com/2182307/144557357-b81b55dd-c750-492f-beea-11e985e2a80b.png">

Reply:
I rebased and pulled in your changes for gasLimit, so now there is a separator between inputs and default and one between default and gasLimi
<img width="281" alt="Screen Shot 2021-12-03 at 4 46 45 PM" src="https://user-images.githubusercontent.com/43930900/144676962-81e301fb-1127-41b7-99fc-eb58f70494e7.png">
t.

Reply:
👍 , can you plz also ensure that margins are as in designs, in above image left and right margin should be 16 px.

Reply:
Image I shared in the above comment is from the design figma. This is the current implementation .

<img width="358" alt="Screen Shot 2021-12-06 at 3 14 31 PM" src="https://user-images.githubusercontent.com/43930900/144915752-e862fcdf-a8c6-4552-8328-d98b30ef36a5.png">